src/python/other-bots/chromium-tests-syncer/run.py

```diff
@@ -85,7 +89,9 @@ def unpack_crash_testcases(crash_testcases_directory):
 
     # 8. Skip in-process fuzzer testcases, since these are only applicable to
     # fuzz targets and don't run with blackbox binaries.
-    if testcase.fuzzer_name and testcase.fuzzer_name in ['afl', 'libFuzzer']:
+    if testcase.fuzzer_name and testcase.fuzzer_name in [
```

Comment:
We can definitely push the filtering down to datastore and avoid iterating through obviously-useless entries.

You'll want to use `Testcase.query`, like: https://github.com/google/clusterfuzz/blob/ad6131e9d61cdbe7838d26b55d82200faeddf84f/src/local/butler/scripts/backfiler.py#L31-L35

Reply:
That'd be great. Will try in a follow up.